content/applications/finance/expenses.rst

```diff
@@ -336,49 +408,72 @@ there are four options:
 approve multiple expense reports at once, remain in the list view. First, select the reports to
 approve by clicking the check box next to each report, or click the box next to :guilabel:`Employee`
-to select all reports in the list. Next, click on the :guilabel:`⚙️ Action (gear)` icon, then click
-:guilabel:`Approve Report`.
+to select all reports in the list.
+
+.. important::
+   Only reports with a status of :guilabel:`Submitted` can be approved. It is recommended to only
+   display the submitted reports by adjusting the status filter on the left side by only having the
+   :guilabel:`Submitted` filter checked off.
+
+   If a report that is unable to be approved is selected, the :guilabel:`Approve Report` button
+   **will not appear**, indicating there is a problem with the selected report(s).
+
+Next, click the :guilabel:`Approve Report` button.
 
 .. image:: expenses/approve-report.png
    :align: center
    :alt: Approve multiple reports by clicking the checkboxes next to each report.
 
 To approve an individual report, click on a report to go to a detailed view of that report. In this
-view, several options are presented: :guilabel:`Approve`, :guilabel:`Refuse`, or :guilabel:`Reset to
-draft`. Click :guilabel:`Approve` to approve the report.
+view, several options are presented: :guilabel:`Approve`, :guilabel:`Report in Next Payslip`,
+:guilabel:`Refuse`, or :guilabel:`Reset to draft`. Click :guilabel:`Approve` to approve the report.
 
 If :guilabel:`Refuse` is clicked, a pop-up window appears. Enter a brief explanation for the refusal
-in the :guilabel:`Reason to refuse Expense` field, then click :guilabel:`Refuse`.
+in the :guilabel:`Reason to Refuse Expense` field, then click :guilabel:`Refuse`.
 
 .. image:: expenses/refuse-expense.png
    :align: center
    :alt: Send messages in the chatter.
 
 Team managers can easily view all the expense reports for their team members. While in the
-:guilabel:`Reports to Approve` view, click on :guilabel:`Filters`, then click :guilabel:`My Team`.
-This presents all the reports for the manager's team.
+:guilabel:`Expense Reports` view, click the drop-down arrow in the right-side of the search box, and
+click on :guilabel:`My Team` in the :guilabel:`Filters` section. This presents all the reports for
+the manager's team.
 
 .. image:: expenses/my-team-filter.png
    :align: center
    :alt: Select the My Team filter.
 
 .. note::
    If more information is needed, such as a receipt is missing, communication is easy from the
-   chatter. In an individual report, simply type in a message, tagging the proper person (if
-   needed), and post it to the chatter by clicking :guilabel:`Send`. The message is posted in the
-   chatter, and the person tagged will be notified via email of the message, as well as anyone
-   following.
+   chatter. In an individual report, simply click :guilabel:`Send message` to open the message text
```

Comment:
Should _message_ be capitalized here? Is it on the UI?
```suggestion
   chatter. In an individual report, simply click :guilabel:`Send message` to open the message text
```

Reply:
It's the box that pops up that allows you to type in a message- but the box itself isn't labeled as anything, so I am using the term "message box" generically. I am not sure if there is a better word/description for it but that made the most sense to me. I left it lower case because that is not the proper name for it, and you don't see those words on the UI.